Runtime/Scripts/Schema/Node.cs

```diff
@@ -77,5 +77,17 @@ public class NodeExtensions {
     [System.Serializable]
     public class Extras {
```

Comment:
The Extras data class is used both for individual Nodes and for the root. This is a bit messy, but I don't see a way to create data classes with different names that parse properties with the same name in different contexts...